training/_posts/2016-10-18-scala-intro.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to Scala Online
+description: Hands-on course, scheduled over 30 days, 43 lectures, 8 hours of video, weekly office hours (videoconference), individual support
+link-out: https://www.getscala.com/#live
+where: Online, world-wide, office hours in multiple time zones
```

Comment:
Shorten to just "online" else it will break the formatting. This field is not meant to wrap.